### PR TITLE
Fixes #68 by adding verbose flag. Also added some logging to walker

### LINKED ISSUE #68: Mockery has no logging to debug it
n/t
(working on a patch)

### PR DESCRIPTION
**Pull request by gaffo**
fixes #68

### CODE REVIEW

File: cmd/mockery/mockery.go [hunk 1]
```diff
@@ -66,25 +75,30 @@ func main() {
...
 		}
 	}
+	log.Printf("Set OutputProvider to %s", osp)
 
 	visitor := &mockery.GeneratorVisitor{
 		InPackage: config.fIP,
 		Note:      config.fNote,
 		Osp:       osp,
 	}
+	log.Printf("Using Wisitor: %s", visitor)
```
Comment (evanphx) [inline cmd/mockery/mockery.go]: Typo? Should be `Visitor`?

Comment (gaffo) [inline cmd/mockery/mockery.go]: yes

**Comment by gaffo** [comment]
Fixed

**Comment by evanphx** [comment]
Looking at this PR again, it's really way too much output that is always printed. Terminals will be full of log message constantly, I can't merge this unless all of that output is behind some kind of debug flag.

**Pull request closed without merging**